CONTRIBUTING.md

```diff
@@ -84,31 +117,39 @@ Detailed description of your event goes here. You can use markdown formatting.
 
 ## Development Guidelines
 
-### Setting Up Your Environment
+### → Setting Up Your Environment
 
 1. Clone the repository
+
+```bash
+git clone https://github.com/YOUR-USERNAME/maintainermonth.git
+```
```

Comment:
Indentation is important here. If there is no indentation, after this point, the list will restart from no. 1.
You can use [Markdown Live Preview](https://markdownlivepreview.com/) to check on the result.

```suggestion
    ```bash
    git clone https://github.com/YOUR-USERNAME/maintainermonth.git
    ```
```